### PR TITLE
[MARKENG-430] Fix 3xx external links

### PR DESCRIPTION
**Pull request by pamelagilmour**
##Tickets/Issues

[MARKENG-430]

##Root Cause

Urls throwing 300 status codes

##Solution Description

Updated urls from the spreadsheet with resolved urls

[MARKENG-430]: https://postmanlabs.atlassian.net/browse/MARKENG-430

### CODE REVIEW

**Review by jkonrath-postman — COMMENTED** [review]
The legacy-documentation ones aren't a worry, since they're not built.

@tristandenyer These look ok to me but I wasn't sure about the long-term maintenance that might result.

Example: If you change https://docs.postman-echo.com/ to the redirect (https://www.postman.com/postman/workspace/published-postman-templates/documentation/631643-f695cab7-6878-eb55-7943-ad88e1ccfd65?ctx=documentation) won't that break if they ever move the target? Seems like it would make more sense to link to the first one, but I'm not an SEO person, so I'm not sure...

**Comment by tristandenyer** [comment]
> If you change https://docs.postman-echo.com/ to the redirect (https://www.postman.com/postman/workspace/published-postman-templates/documentation/631643-f695cab7-6878-eb55-7943-ad88e1ccfd65?ctx=documentation) won't that break if they ever move the target? Seems like it would make more sense to link to the first one, but I'm not an SEO person, so I'm not sure...

I see your point @jkonrath-postman . I am just not sure how long the redirect for https://docs.postman-echo.com/ will live. To depend on a redirect is a bit brittle, even thought the URL for the new page appears brittle. We have an external link check step in the process (Frog), so if the new one breaks / moves, we will catch it then.

I'd like to stick to the Rule of always point to the resolved link, lest there be a reason (temporary, or otherwise.) This is in part SEO, but also UX, and moreover less reliance on middleware.

**Review by tristandenyer — APPROVED** [review]
LGTM, will also run Frog when in Beta.

File: legacy-documentation/v5/postman/team_library/activity_feed_and_restoring_collections.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ You can review a chronological list of activities about all collections shared w
 
 You can also review the activity feed from the Dashboard. 
 
-Log in to the [Postman Website](https://app.getpostman.com) and click the **Dashboard** button. 
+Log in to the [Postman Website](https://postman.postman.co/workspaces) and click the **Dashboard** button.
```
Comment (tristandenyer) [inline legacy-documentation/v5/postman/team_library/activity_feed_and_restoring_collections.md:40]: @pamelagilmour Let's change this one to be `https://go.postman.co. The new link is pointing to our Team's workspace (you were likely logged in when resolving the link.)

FYI @jkonrath-postman

File: src/components/Header/Header.data.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
       "url": "/docs/developer/resources-intro/"
     },{
       "name": "Dashboard",
-      "url": "https://app.getpostman.com/",
+      "url": "https://postman.postman.co/workspaces",
```
Comment (tristandenyer) [inline src/components/Header/Header.data.json:14]: @pamelagilmour Let's change this one to be `https://go.postman.co. The new link is pointing to our Team's workspace (you were likely logged in when resolving the link.)

File: src/pages/docs/collaborating-in-postman/commenting-on-collections.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ You can leave comments on collections from Postman.
 
 > Team members must have access to the collection in question in order to post comments.
 
-1. Sign in to [Postman](https://app.getpostman.com).
+1. Sign in to [Postman](https://postman.postman.co/workspaces).
```
Comment (tristandenyer) [inline src/pages/docs/collaborating-in-postman/commenting-on-collections.md:38]: @pamelagilmour Let's change this one to be `https://go.postman.co. The new link is pointing to our Team's workspace (you were likely logged in when resolving the link.)

File: src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md [hunk 4]
```diff
@@ -82,11 +82,11 @@ To filter by entity, click **Entity** at the top of the activity feed and select
 
 ## Viewing team activity
 
-You can review the activity for a team with a Postman Team, Business, or Enterprise account. In [Postman](https://app.getpostman.com), use the __Workspaces__ dropdown to select your team, then navigate to the __Activity__ feed to view the events.
+You can review the activity for a team with a Postman Team, Business, or Enterprise account. In [Postman](https://postman.postman.co/workspaces), use the __Workspaces__ dropdown to select your team, then navigate to the __Activity__ feed to view the events.
```
Comment (tristandenyer) [inline src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md:85]: @pamelagilmour Let's change this one to be `https://go.postman.co. The new link is pointing to our Team's workspace (you were likely logged in when resolving the link.)

File: src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md [hunk 5]
```diff
@@ -82,11 +82,11 @@ To filter by entity, click **Entity** at the top of the activity feed and select
...
 
 ## Viewing user activity
 
-You can review the activity for your own account in [Postman](https://app.getpostman.com). Click your avatar in the top right, and select __Profile__ — the __Activity__ tab will list your events.
+You can review the activity for your own account in [Postman](https://postman.postman.co/workspaces). Click your avatar in the top right, and select __Profile__ — the __Activity__ tab will list your events.
```
Comment (tristandenyer) [inline src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md:89]: @pamelagilmour Let's change this one to be `https://go.postman.co. The new link is pointing to our Team's workspace (you were likely logged in when resolving the link.)

File: src/pages/docs/getting-started/installation-and-updates.md [hunk 6]
```diff
@@ -168,7 +168,7 @@ The native app is built on [Electron](https://www.electronjs.org/), and [overco
 
 ### Migrating to the native app
 
-To switch from the Chrome app to native, [download](https://www.postman.com/downloads/) Postman and [sign in to your account](https://app.getpostman.com/). Start the native app, and your history and collections will be automatically synced.
+To switch from the Chrome app to native, [download](https://www.postman.com/downloads/) Postman and [sign in to your account](https://postman.postman.co/workspaces). Start the native app, and your history and collections will be automatically synced.
```
Comment (tristandenyer) [inline src/pages/docs/getting-started/installation-and-updates.md:171]: @pamelagilmour Let's change this one to be `https://go.postman.co. The new link is pointing to our Team's workspace (you were likely logged in when resolving the link.)

**Review by tristandenyer — CHANGES_REQUESTED** [review]
@pamelagilmour Let's change these 6 links

**Comment by pamelagilmour** [comment]
[MARKENG-430] Updated urls accordingly

**Review by tristandenyer — APPROVED** [review]
LGTM

**Pull request merged**